app/dashboard/sync/tezos.py

```diff
@@ -81,7 +78,5 @@ def sync_tezos_payout(fulfillment):
             fulfillment.accepted_on = timezone.now()
             fulfillment.accepted = True
             record_payout_activity(fulfillment)
-        elif txn_status == 'expired':
-            fulfillment.payout_status = 'expired'
```

Comment:
^ why are we removing expired ?

Reply:
is a failed tx same as expired?

the notice to the user seems misleading -> ```We couldn't find the transaction on chain. Please ensure the payment has been made and click on payout bounty```

Reply:
but this means the txn will forever be in pending ! 
I'd say we should failed and expired simply cause it tells the user -> that your txn was not successful -> try again